vowpalwabbit/options.h

```diff
@@ -152,6 +153,24 @@ struct options_i
     return dynamic_cast<const typed_option<T>&>(base);
   }
 
+  // Check if option values exist and match.
+  // Add if it does not exist.
+  template<typename T>
+  bool add_or_check_options(const std::string& name, T expected_val){
+    if(was_supplied(name)) {

```

Comment:
Formatting does not comply with clang format

Reply:
Changed